lib/tsconfig/4.6.x/tsconfig.json

```diff
@@ -0,0 +1,72 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "allowUmdGlobalAccess": false,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "alwaysStrict": true,
+    "charset": "utf8",
+    "declaration": true,
+    "diagnostics": false,
+    "downlevelIteration": true,
+    "emitBOM": false,
+    "emitDecoratorMetadata": true,
+    "emitDeclarationOnly": false,
+    "exactOptionalPropertyTypes": false,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": false,
+    "importHelpers": true,
+    "incremental": true,
+    "inlineSourceMap": false,
+    "inlineSources": true,
+    "isolatedModules": false,
+    "listEmittedFiles": false,
+    "listFiles": false,
+    "module": "es2020",
+    "moduleResolution": "node",
+    "noEmitHelpers": false,
+    "noEmitOnError": false,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitOverride": true,
+    "noImplicitUseStrict": false,
+    "noPropertyAccessFromIndexSignature": true,
+    "noUncheckedIndexedAccess": false,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "preserveConstEnums": true,
+    "pretty": true,
+    "removeComments": false,
+    "sourceMap": true,
+    "strict": true,
+    "strictBindCallApply": true,
+    "strictFunctionTypes": true,
+    "strictPropertyInitialization": true,
+    "strictNullChecks": true,
+    "stripInternal": true,
+    "skipLibCheck": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "suppressExcessPropertyErrors": false,
+    "useDefineForClassFields": true,
+    "useUnknownInCatchVariables": true,
+    "target": "es2020",
```

Comment:
Could you add a comment about this change (previously it was es2015) in your commit message ?

Does the Angular recommends/forces to use the targert `es2020` ?